### ISSUE TITLE
Act based on file type

### ISSUE THREAD

**Issue by parsanoori** [tfeldmann/organize#106]
On Hazel I love to sort my downloads based on **Kind**, is there a way by which I can do it with "organize" too ?

**Comment by quintrino** [comment]
Hi @parsanoori, do you mean by filetype? 

If so you can definitely do that with organize. Here's an example which will move all mp4s into videos and play all mp3s with vlc and then move them to audio. 

```
  - folders: ~/Downloads/
    filters:
      - extension: mp4
    actions:
      - move: ~/Downloads/videos/

  - folders: ~/Downloads/
    filters:
      - extension: mp3
    actions:
      - shell: 'vlc --play-and-exit "{path}"'
      - move: ~/Downloads/audio/
      ```

**Comment by tfeldmann** [comment]
@quintrino is right, often filtering by extension works firne. I'm in the process of adding a MIME type filter for this purpose so you can filter by videos / images / application data (Pull request #109)

**Comment by tfeldmann** [comment]
This is now included in the newest version v1.10.

https://organize.readthedocs.io/en/latest/page/filters.html#mimetype
